metricbeat/autodiscover/builder/hints/metrics_test.go

```diff
@@ -368,6 +368,33 @@ func TestGenerateHints(t *testing.T) {
 				},
 			},
 		},
+		{
+			message: "Named port in the hints should return the corresponding container port",
+			event: bus.Event{
+				"host":      "1.2.3.4",
+				"port":      3306,
+				"port_name": "some",
```

Comment:
If `port_name` is going to be included in all events we should probably add a `port_name` to the rest of test cases.
Or at least add a test case with an event with `port_name`, but with a hosts hint that doesn't use named ports.